pkg/client/example-runner.go

```diff
@@ -55,10 +68,17 @@ func RunExamplesForService(registry broker.BrokerRegistry, client *Client, servi
 
 }
 
+type CompleteServiceExample struct {
+	Example                    broker.ServiceExample   `json: "service_example"`
```

Comment:
I think you should embed this reference so all the example's input fields are on the same level.

To get the JSON to be nice, you can add the tag `json:",inline"` which will embed all the child fields at the root level.

Reply:
Modified struct to:
```
type CompleteServiceExample struct {
	Example        broker.ServiceExample   `json: "service_example, inline"`
	ServiceName    string                  `json: "service_name, inline"`
	ServiceId      string                  `json: "service_id, inline"`
	ExpectedOutput []broker.BrokerVariable `json: "broker_variables, inline"`
}
```